pkg/kubernetes/version.go

```diff
@@ -0,0 +1,220 @@
+package kubernetes
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	// https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
+	AnnotationSpinnakerArtifactVersion = `artifact.spinnaker.io/version`
+	AnnotationSpinnakerMonikerSequence = `moniker.spinnaker.io/sequence`
+	Annotation
+)
+
+type SpinnakerVersion struct {
+	Long  string
+	Short string
+}
+
+//Create a similar function for labels
```

Comment:
Remove this comment. Add in its place a descriptive comment about the function that starts with the function name. Example [here](https://github.com/gin-gonic/gin/blob/master/logger.go#L81).